src/test/librados/io_cxx.cc

```diff
@@ -218,6 +218,39 @@ TEST_F(LibRadosIoPP, SparseReadOpPP) {
   }
 }
 
+TEST_F(LibRadosIoPP, SparseReadExtentArrayOpPP) {
+  int buf_len = 32;
```

Comment:
Not following why `buf_len` was silently reduced from 128 to 32.

Reply:
Sorry forgot to mention this.
The larger buffer length the longer time it will take for the test. The length here won't affect the test anyway.